frontend/src/metabase/dashboard/hooks/use-register-dashboard-metabot-context.ts

```diff
@@ -0,0 +1,16 @@
+import { useRegisterMetabotContextProvider } from "metabase/metabot";
+
+import { getDashboard } from "../selectors";
+
+export const useRegisterDashboardMetabotContext = () => {
+  useRegisterMetabotContextProvider((state) => {
+    const dashboard = getDashboard(state);
+    if (!dashboard) {
+      return {};
+    }
+
+    return {
+      user_is_viewing: [{ type: "dashboard", id: dashboard.id }],
```

Comment:
@Somtom As we discussed, I think `user_is_viewing` isn't used anymore. Would it make sense to remove FE code related to that? Do you plan to re-introduce it later on?

Reply:
I'll comment on the other comment 😁